src/libtriton/arch/x86/x86Semantics.cpp

```diff
@@ -326,22 +328,35 @@ VMOVD                        | avx        | VEX Move Doubleword
 VMOVDQA                      | avx        | VEX Move aligned packed integer values
 VMOVDQU                      | avx        | VEX Move unaligned packed integer values
 VMOVQ                        | avx        | VEX Move Quadword
+VMOVSD                       | avx        | VEX Move or Merge Scalar Double-Precision Floating-Point Value
+VMOVAPS                      | avx        | VEX Move Aligned Packed Single-Precision Floating-Point Values
+VMOVUPS                      | avx        | VEX Move Unaligned Packed Single-Precision Floating-Point Values
 VPAND                        | avx/avx2   | VEX Logical AND
 VPANDN                       | avx/avx2   | VEX Logical AND NOT
 VPEXTRB                      | avx/avx2   | VEX Extract Byte
 VPEXTRD                      | avx/avx2   | VEX Extract Dword
 VPEXTRQ                      | avx/avx2   | VEX Extract Qword
 VPEXTRW                      | avx/avx2   | VEX Extract Word
+VPBROADCASTB                 | avx2       | VEX Load Byte Integer and broadcast
```

Comment:
?

Reply:
Forgot to add description for this instruction in 6d12775163284ba0309204344415c3d52caecb28

Reply:
Indeed just saw that, i was thinking you missed a semantic because i've not seen the function VPBROADCASTB in your patch :).